### PR TITLE
[TW-765] doc viewers can pause monitors

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 1]
```diff
@@ -216,19 +216,21 @@ Move mock server | &#x2714; |
 
 You can [assign](/docs/monitoring-your-api/faqs-monitors/#who-can-edit-my-monitors) two role types for Postman Monitors: **Editor** and **Viewer**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:217]: Does this need to be "You can assign four role types..." based on the new Super Admin and Admin roles?

**Review by mark-anthony-postman — APPROVED** [review]
Looks good! 🌟 Had one question for you

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:217]: Good catch!

**Pull request merged**